### PR TITLE
docs(example): sanity added

### PR DESCRIPTION
**Pull request by Anishpras**

### CODE REVIEW

File: examples/sanity/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Sanity Example!
```
Comment (kentcdodds) [inline examples/sanity/README.md:1]: Could you update this to follow the format in the `examples/template/README.md` please?

File: examples/sanity/app/styles/index.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+/*
```
Comment (kentcdodds) [inline examples/sanity/app/styles/index.css:1]: Could you remove irrelevant files like this one? Try to keep examples as minimal as possible (as noted in [the instructions](https://github.com/remix-run/remix/tree/main/examples#contributing)).

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Thanks for this! It seems like this example might be a bit bigger than we normally want our examples to be. Is there a way to minimize it to something much simpler?

Also, we want our examples to be TypeScript. Can you rewrite everything to TypeScript in the example?

File: examples/sanity/.npmrc [hunk 3]
```diff
@@ -0,0 +1 @@
+@remix-run:registry=https://npm.remix.run
```
Comment (MichaelDeBoey) [inline examples/sanity/.npmrc:1]: This file can be deleted

File: examples/sanity/app/entry.server.jsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import ReactDOMServer from "react-dom/server";
+import { RemixServer } from "remix";
+import dotenv from 'dotenv'
```
Comment (MichaelDeBoey) [inline examples/sanity/app/entry.server.jsx:3]:
```suggestion

import 'dotenv/config';
```

File: examples/sanity/app/entry.server.jsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+export default function handleRequest(
+  request,
+  responseStatusCode,
+  responseHeaders,
+  remixContext
+) {
+  dotenv.config()
+
```
Comment (MichaelDeBoey) [inline examples/sanity/app/entry.server.jsx:12]:
```suggestion
```

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

**Comment by kentcdodds** [comment]
Additionally, since this was pretty much copied from @simeonGriggs' [guide on sanity](https://www.sanity.io/guides/remix-run-live-preview) we should include a link to the original guide in the README>

**Comment by Anishpras** [comment]
> Additionally, since this was pretty much copied from @SimeonGriggs' [guide on sanity](https://www.sanity.io/guides/remix-run-live-preview) we should include a link to the original guide in the README.

Yes, I have done all the necessary changes and included the link to the original guide.

**Comment by kentcdodds** [comment]
Whoops, unfortunately when I switched the branch from `dev` to `main` (examples should be made against the `main` branch not `dev`) that kinda messed stuff up a bit. I can clean it up for you later if you don't get to it first 😅

**Review by kentcdodds — APPROVED** [review]
Looks awesome. Thanks! Sorry again for messing you up...

**Pull request merged**